CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -599,6 +603,19 @@ local function MakeSlot(parent, index)
 	SlotFrame.MouseButton1Click:connect(function() changeSlot(slot) end)
 	slot.Frame = SlotFrame
 
+	do
+		local selectionObjectClipper = NewGui('Frame', 'SelectionObjectClipper')
+		selectionObjectClipper.Visible = false
+		selectionObjectClipper.Parent = SlotFrame
+
+		SelectionObj = NewGui('ImageLabel', 'Selector')
+		SelectionObj.Size = UDim2.new(1, 0, 1, 0)
+		SelectionObj.Image = "rbxasset://textures/ui/Keyboard/key_selection_9slice.png";
```

Comment:
They're semiCOOLons, I think they just got left in there b/c I a copied pasta'd this data from one of those Util.Create thingies. I shall remove them!